pom.xml

```diff
@@ -0,0 +1,245 @@
+    <name>Gravitee.io - Tracing - OpenTelemetry OTLP</name>
+
+    <properties>
+        <gravitee-bom.version>8.1.0</gravitee-bom.version>
+        <gravitee-node-api.version>5.18.0</gravitee-node-api.version>
+        <opentelemetry-instrumentation-bom-alpha.version>1.33.3-alpha</opentelemetry-instrumentation-bom-alpha.version>
+        <opentelemetry-instrumentation-api.version>1.33.3</opentelemetry-instrumentation-api.version>
+        <awaitility.version>4.2.0</awaitility.version>
+        <testcontainers.version>1.19.3</testcontainers.version>
+        <vertx.version>4.5.7</vertx.version>
+        <maven-plugin-assembly.version>3.6.0</maven-plugin-assembly.version>
+        <maven-plugin-javadoc.version>3.5.0</maven-plugin-javadoc.version>
+
+        <!-- Property used by the publication job in CI-->
+        <publish-folder-path>graviteeio-apim/plugins/tracers</publish-folder-path>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <!-- Import bom to properly inherit all dependencies -->
+            <dependency>
+                <groupId>io.gravitee</groupId>
+                <artifactId>gravitee-bom</artifactId>
+                <version>${gravitee-bom.version}</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers-bom</artifactId>
+                <version>${testcontainers.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>io.opentelemetry.instrumentation</groupId>
+                <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
+                <version>${opentelemetry-instrumentation-bom-alpha.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <!-- Gravitee dependencies -->
+        <dependency>
+            <groupId>io.gravitee.node</groupId>
+            <artifactId>gravitee-node-api</artifactId>
+            <scope>provided</scope>
+            <version>${gravitee-node-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.gravitee.node</groupId>
+            <artifactId>gravitee-node-vertx</artifactId>
+            <scope>provided</scope>
+            <version>${gravitee-node-api.version}</version>
+        </dependency>
+
+        <!-- Vert.x dependencies -->
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-core</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-grpc-client</artifactId>
+            <version>${vertx.version}</version>
```

Comment:
Isn't already declared in gravitee-bom>vertx-dependencies ?

Reply:
No, it's not @phiz71

Reply:
https://github.com/vert-x3/vertx-dependencies/blob/4.5.7/pom.xml#L911-L914 ?